Source/SvgDocument.cs

```diff
@@ -23,6 +23,13 @@ namespace Svg
     /// </summary>
     public class SvgDocument : SvgFragment, ITypeDescriptorContext
     {
+        /// <summary>
+        /// Skip check whether the GDI+ can be loaded.
+        /// </summary>
+        /// <remarks>
+        /// Set to true on systems that do not support GDI+ like UWP.
+        /// </remarks>
+        public static bool SkipGDIPlusCapabilityCheck = false;

```

Comment:
Is if-def unnecessary ?
```cs
#if NETSTANDARD20
public static bool SkipGDIPlusCapabilityCheck = true;
#else
public static bool SkipGDIPlusCapabilityCheck = true;
#endif
```

Reply:
Yep, I suggest keeping the check on false unless explicitly set to false by the caller. We should for now assume that the callee will use some of the graphical elements, thus requiring us to check the GDI capability. If the callee is absolutely sure to not use the graphic elements he can ommit the check